db/migrate/20220912103735_create_seances.rb

```diff
@@ -0,0 +1,13 @@
+class CreateSeances < ActiveRecord::Migration[7.0]
+  def change
+    create_table :seances do |t|
+      t.datetime :start_time
+      t.datetime :end_time
+      t.float :price
```

Comment:
I think it would be better if those also had `null: false`, WDYT?

Reply:
I also think that it's not the best idea to store price as a float. Floating point numbers store their values in binary base system (0s and 1s) which means most usual values will have approximation errors.

Instead of using float you can consider using decimal (which doesn't use binary base to store their values) or integer. For example 10.99 PLN can be saved as 1099 using integer. You can read more about that [here](https://dev.to/songta17/how-to-handle-a-price-with-ruby-on-rails-54a0).